supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -396,6 +396,17 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*: xref:app[app]
 
+[discrete]
+[[application-stream]]
+==== image:images/yes.png[yes] Application Stream (noun)
+*Description*: _Application Streams_ are multiple versions of RHEL user-space components that are delivered and updated more frequently than the core operating system packages. Application Streams can be packaged as RPMs packages, modules, or Software Collections. Do not confuse with AppStream, which is the repository through which Application Streams are distributed. Note that the AppStream repository contains also components other than Application Streams.
```

Comment:
```suggestion
*Description*: _Application Streams_ are multiple versions of RHEL user-space components that are delivered and updated more frequently than the core operating system packages. Application Streams can be packaged as RPMs packages, modules, or Software Collections. Do not confuse with AppStream, which is the repository through which Application Streams are distributed. Note that the AppStream repository also contains components other than Application Streams.
```

Reply:
I think the word 'also' should be switched like this so that the sentence reads correctly.